docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -404,6 +404,64 @@ your application's `web.xml` file, as shown in the example below:
 </web-app>
 ----
 
+== Logout Configuration
+
+OpenID Connect allows the client to configurate application based URIs that will
+trigger the OpenID Provider to log out the user.
+
+The configuration for this feature can be specified in the application's
+`oidc.json` file or in the `secure-deployment` resource of the `elytron-oidc-client` subsystem.
+
+Here is an example of an `oidc.json` configuration file:
+
+[source,options="nowrap"]
+----
+{
+  "client-id" : "customer-portal",
+  "provider-url" : "http://localhost:8180/auth/realms/demo",
+  "ssl-required" : "external",
+  "logout-path" : "portal/logout",
+  "logout-callback-path" : "http://localhost:8080/customer-portal/goodbye-page",
+   "credentials" : {
+      "secret" : "234234-234234-234234"
+   }
+}
+----
+
+* `logout-path`: The path component of a URL.  It is uses to identify the URL that is used to start an OIDC logout request.
+* `logout-callback-path`: An absolute URI.  It will be called after logout has been executed.  It is used by Front-channel and Back-channel logout.  The URI must be registered with the OpenId Provider.
+* `post-logout-redirect-uri` : An absolute URI.  It will be called after logout has been executed.  It is used by RP-Initiated logout only.
+ The URI must be registered with the OpenId Provider.
+* `logout-session-required` :  When set to `true` the `sid` and `iss` parameters will be enforced when processing front-channel logout requests and the `sid` will be enforced when processing back-channel requests.  If set to `false` these parameters will not be checked and any authenticated request will invalidate the local user session.
```

Comment:
We should also document the session invalidation option.

Reply:
done